client/src/features/calendarHeader/CalendarHeader.jsx

```diff
@@ -19,7 +19,9 @@ function CalendarHeader({ date }) {
   const DISCORD_THREAD_URL =
     "discord://discord.com/channels/735923219315425401/1038482732633825442";
 
-  const GH_ISSUES_URL = "https://github.com/Together-100Devs/Together/issues";
+  const GH_DISCUSSIONS_URL =
+    "https://github.com/Together-100Devs/Together/discussions";
+  const GH_WIKI_URL = "https://github.com/Together-100Devs/Together/wiki";
```

Comment:
we need to change this now as our docs location has changed to https://github.com/Together-100Devs/Together/tree/main/docs

so add this link instead please: https://github.com/Together-100Devs/Together/tree/main/docs